src/converter/frame.py

```diff
@@ -39,9 +52,95 @@ def post_process_frame(fig_frame: dict, sketch_frame: Frame) -> Frame:
     if sketch_frame.overlaySettings is not None:
         sketch_frame.layers.insert(0, rectangle.make_clipping_rect(fig_frame, sketch_frame.frame))
 
+    # Figma stores its stack children in bottom up order, but Sketch uses top down
+    if utils.has_auto_layout(fig_frame):
+        # If the layout has a child which is ignoring the Stack layout, and the stack
+        # has a "Last on top" z-index order, we'll remove the stack layout.
+        has_child_ignoring_layout = False
+
+        for layer in sketch_frame.layers:
+            if (
+                hasattr(layer, "flexItem")
+                and layer.flexItem
+                and getattr(layer.flexItem, "ignoreLayout", False)
+            ):
+                has_child_ignoring_layout = True
+                break
+
+        if has_child_ignoring_layout and not fig_frame.get("stackReverseZIndex"):
+            sketch_frame.groupLayout = FreeFormGroupLayout()
+            utils.log_conversion_warning("STK001", fig_frame)
+        else:
+            sketch_frame.layers.reverse()
```

Comment:
As per the comment, Sketch doesn't support "last on top" z-index ordering. And so if we encounter a stack with `stackReverseZIndex` set to `false` and one of the layout's children is positioned absolutely (ignores the stack layout), then we opt to simply not convert the stack and prioritise visual parity over auto layout conversion.